### PR TITLE
Acq id bash

### LINKED ISSUE #68: Update file-transfer

### LINKED ISSUE #69: Update floppy disk transfer

### LINKED ISSUE #70: Update optical media transfer

### LINKED ISSUE #71: Update iomega disk transfer

### PR DESCRIPTION
**Pull request by nkrabben**
Immediate changes made to all transfer pages that use bash scripts or commands.
Resolves #68, Resolves #69, Resolves #70, and Resolves #71

Swapped in ACQ_acqID_specObjectID wherever it made sense and changed mentions of collection to acquisition.

I think the whole transfer section could use a good readthrough to find and replace collection with acquisition. Better to hold off on this while considering wholesale rewrites to multiple pages since we spend a lot of time repeating instructions between Mac and WSL and different transfer methods.

### CODE REVIEW

File: transfers/file-transfers.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ or
 * Use ``mkdir`` command to create directories when media aren't consecutively numbered:
 
   * Enter ```mkdir``` command.  
-```mkdir -p CollID/Media-000{1..9}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-00{10..99}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-000{1,5,7,9}/{metadata/submissionDocumentation,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```
```
Comment (arhonemus) [inline transfers/file-transfers.md:60]: ```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID/{metadata,objects}```
```suggestion
```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```  
```

File: transfers/file-transfers.md [hunk 2]
```diff
@@ -57,9 +57,9 @@ or
 * Use ``mkdir`` command to create directories when media aren't consecutively numbered:
 
   * Enter ```mkdir``` command.  
-```mkdir -p CollID/Media-000{1..9}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-00{10..99}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-000{1,5,7,9}/{metadata/submissionDocumentation,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{10..99}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1,5,7,9}/{metadata,objects}```
```
Comment (arhonemus) [inline transfers/file-transfers.md:62]:
```suggestion
```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{10..99}/{metadata,objects}```  
```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1,5,7,9}/{metadata,objects}```  
```

File: transfers/file-transfers.md [hunk 3]
```diff
@@ -57,9 +57,9 @@ or
 * Use ``mkdir`` command to create directories when media aren't consecutively numbered:
 
   * Enter ```mkdir``` command.  
-```mkdir -p CollID/Media-000{1..9}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-00{10..99}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-000{1,5,7,9}/{metadata/submissionDocumentation,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{10..99}/{metadata,objects}```
```
Comment (arhonemus) [inline transfers/file-transfers.md]:
```suggestion
```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID/{metadata,objects}```  
```

File: transfers/file-transfers.md [hunk 4]
```diff
@@ -57,9 +57,9 @@ or
 * Use ``mkdir`` command to create directories when media aren't consecutively numbered:
 
   * Enter ```mkdir``` command.  
-```mkdir -p CollID/Media-000{1..9}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-00{10..99}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-000{1,5,7,9}/{metadata/submissionDocumentation,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{10..99}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1,5,7,9}/{metadata,objects}```
```
Comment (arhonemus) [inline transfers/file-transfers.md:72]:
```suggestion
  
```

File: transfers/file-transfers.md [hunk 5]
```diff
@@ -83,15 +83,14 @@ Or
 * Change to fileTransfers directory.
 ```$ cd /Volumes/DigArchDiskStation/Staging/ingest/fileTransfers```
   * Enter ```mkdir``` command.  
-```mkdir -p CollID/Media-000{1..9}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-00{10..99}/{metadata/submissionDocumentation,objects}```  
-```mkdir -p CollID/Media-000{1,5,7,9}/{metadata/submissionDocumentation,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1..9}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{10..99}/{metadata,objects}```  
+```mkdir -p ACQ_acqID/ACQ_acqID_specObjectID{1,5,7,9}/{metadata,objects}```
```
Comment (arhonemus) [inline transfers/file-transfers.md:97]:
```suggestion

```

**Review by arhonemus — CHANGES_REQUESTED** [review]
remove bracketed numbers in the mkdir command across pages.

**Comment by nkrabben** [comment]
> remove bracketed numbers in the mkdir command across pages.

removed

**Review by arhonemus — APPROVED** [review]

**Pull request merged**